protected/humhub/modules/user/migrations/m201025_095247_spaces_of_users_group.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use yii\db\Migration;
+
+/**
+ * Class m201025_095247_spaces_of_users_group
+ */
+class m201025_095247_spaces_of_users_group extends Migration
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function safeUp()
+    {
+        $this->createTable('group_spaces', [
```

Comment:
Please do not use a plural here. `group_space`

Reply:
Done!